src/components/Header.tsx

```diff
@@ -8,7 +12,18 @@ type HeaderProps = {
 
 export default function Header(props: HeaderProps) {
   const { setSidebarOpen } = props;
-  // const [notifications, setNotifications] = useState(false);
+  const [info, setInfo] = useState<Info | null>(null);
+
+  useEffect(() => {
+    getInfo()
+      .then((info: Info) => {
+        setInfo(info);
```

Comment:
TODO: we should prob move this into it's own InfoContext or other storage mechanism since its now used by two components

Reply:
Great thinking, agreed